### ISSUE TITLE
[v2 QUESTION]: How to use the --atom-features-path /path/to/atom/features.npz in the case of molecule+solvent

### ISSUE THREAD

**Issue by xiaoboy185** [chemprop/chemprop#1027]
**What are you trying to do?**
I am considering the predict the property of molecule in the solvent, so two molecules should be input into the gnn at the same time.
While i have additionally calculated atom- and bond-related properties for molecules and sovlents. 
I know it is possible to include the atom-features for the case with only one molecule.
I am wondering whether it is possible to include atom- and bond- features for each two molecules by using --atom-features-path /path/to/atom/features.npz command lines. If it is possible, please show me some example for the features.npz file.
**Previous attempts**
If possible, provide some examples of what you've already tried and what the output was.

**Screenshots**

**Comment by KnathanM** [comment]
This is possible and documented [here](https://chemprop.readthedocs.io/en/main/cmd.html). Search for `--atom-features-path`, `--atom-descriptors-path`, and `--bond-features-path`. Let me know if you still have questions after reading that. 

For devs: Looks like we haven't updated this in the CLI tutorials [here](https://chemprop.readthedocs.io/en/main/tutorial/cli/train.html#additional-features). We should probably do that.

**Comment by xiaoboy185** [comment]
Thanks very much! I have sucessfully included the atom features in the calcualtion.
While one problem is i have to create different .npz files for the training and testing process since they are using different molecules. I am wondering whether it is possbile to finished this work within one command line, after that the output files for the predicted values of traing targets, val targets and testing targets, could be obtained.
In v1 version, the predicted traing targets could be saved, while in v2 version,  i can not find the predicted target.

**Comment by KnathanM** [comment]
You should be able to use a single .npz file for all training and testing datapoints. The data in the .npz file is loaded into datapoints before the data is split into train and test. You can specify which datapoints you want to go into train and test using user defined splits. The documentation for this is [here](https://chemprop--1041.org.readthedocs.build/en/1041/tutorial/cli/train.html#train-validation-test-splits). Soon it will be added to the main documentation. 

As for getting the model predictions, the predictions on the test set are saved in the output directory. I think the default path is <cwd/chemprop_training/<name of data>/model_0/test_predictions.csv> but it could be a bit different. The predictions on the training and validation set are not saved, but you could get them for the final model by running `chemprop predict -i data.csv`
